### PR TITLE
Preinstall hooks

### PR DESCRIPTION
**Pull request by Twon**
The included integrity checks workflow will validate github action files

### CODE REVIEW

File: DRAFT.md [hunk 1]
```diff
@@ -1029,7 +1029,11 @@ Otherwise, the interface of the specialization is as defined in [optional].
 Add a new feature-test macro:
 
 ```c++
+<<<<<<< HEAD
```
Comment (jbcoe) [inline DRAFT.md:1032]: Merge conflict markers to be removed.

**Review by jbcoe — CHANGES_REQUESTED** [review]
This looks great! Happy to approve and submit once those merge-markers have gone.

**Comment by Twon** [comment]
> This looks great! Happy to approve and submit once those merge-markers have gone.

Thanks, the markers were removed.  I'll submit once CI finishes building

**Review by jbcoe — APPROVED** [review]

**Comment by jbcoe** [comment]
This is awesome work, thanks @Twon

**Pull request merged**